tests/run_tests.sh

```diff
@@ -250,7 +257,7 @@ function run_individual_tests()
 
 setup_environment
 
-while getopts "h?a:c:d:e:f:i:k:l:m:n:op:q:rs:t:ux" opt; do
+while getopts "h?a:c:d:e:f:i:k:l:m:n:op:Os:q:rs:t:ux" opt; do
```

Comment:
why is there 's' along with the 'O' option?

Reply:
Sorry, I made it wrong here. Fixed. Thanks for pointing out.